src/main/java/mops/controllers/ApplicationController.java

```diff
@@ -162,6 +162,7 @@ public String modul(final KeycloakAuthenticationToken token, @Valid final WebApp
         if (token != null) {
             webApplicationService.removeCurrentModuleFromListOfAvailebleModuleToApplyTo(
                     token, webApplicant, webAddress, model, modules, webCertificate);
+            LOGGER.debug("Saved Infos of Applicant " + token.getName());
```

Comment:
Ist das wirklich, das, was hier passiert? Entweder ist der Log falsch oder die Methode falsch benannt...

Reply:
methoden name ist doof

Reply:
hab den Methoden namen in was sinnvolleres geändert